_posts/2024-08-09-contributing-to-eclipse-che-docs.adoc

```diff
@@ -0,0 +1,102 @@
+---
+title: Contributing to Eclipse Che documentation
```

Comment:
IMO we need a better title ;-)

Reply:
I'll figure out something catchier

Reply:
up to you, if you would like to keep it works as well